ipinfo/batch.go

```diff
@@ -118,6 +126,13 @@ func (c *Client) GetBatch(
 		batchSize = int(opts.BatchSize)
 	}
 
+	// use correct concurrent requests limit; either default or user-provided.
+	if opts.ConcurrentBatchRequestsLimit == 0 {
+		maxConcurrentBatchRequests = batchDefaultConcurrentRequestsLimit
+	} else {
+		maxConcurrentBatchRequests = opts.ConcurrentBatchRequestsLimit
```

Comment:
Does `errg.SetLimit` treat negative numbers as disabling things? Because if this option is negative that's what's gonna be input there.

Reply:
Yes, it disables the limit. Stated in the documentation that
`// A negative value indicates no limit.`